src/js/containers/covid19/SummaryInsightsContainer.jsx

```diff
@@ -43,53 +45,71 @@ const SummaryInsightsContainer = ({
-        const params = {
-            filter: {
-                def_codes: defCodes.map((defc) => defc.code)
-            }
-        };
-        if (activeTab !== 'all') {
-            params.filter.award_type_codes = awardTypeGroups[activeTab];
-        }
-        if (assistanceOnly && activeTab === 'all') {
-            params.filter.award_type = 'assistance';
-        }
         if (defCodes && defCodes.length > 0) {
-            awardAmountRequest.current = fetchAwardAmounts(params);
-            awardAmountRequest.current.promise
-                .then((res) => {
-                    setAwardObligations(res.data.obligation);
-                    setAwardOutlays(res.data.outlay);
-                    setNumberOfAwards(res.data.award_count);
-
-                    /* eslint-disable camelcase */
-                    // set totals in redux, we can use totals elsewhere to calculate unlinked data
-                    const totals = {
-                        obligation: res.data?.obligation,
-                        outlay: res.data?.outlay,
-                        awardCount: res.data?.award_count,
-                        faceValueOfLoan: res.data?.face_value_of_loan
-                    };
+            if (activeTab === 'all' && !assistanceOnly) {
+                if (Object.keys(allAwardTypeTotals).length > 0) {
+                    setAwardOutlays(allAwardTypeTotals.outlay);
+                    setAwardObligations(allAwardTypeTotals.obligation);
+                    setNumberOfAwards(allAwardTypeTotals.awardCount);
 
                     if (spendingByAgencyOnly) {
-                        dispatch(setTotals('SPENDING_BY_AGENCY', totals));
-                    } else if (assistanceOnly) {
-                        dispatch(setTotals('ASSISTANCE', totals));
+                        dispatch(setTotals('SPENDING_BY_AGENCY', allAwardTypeTotals));
                     } else if (recipientOnly) {
-                        dispatch(setTotals('RECIPIENT', totals));
+                        dispatch(setTotals('RECIPIENT', allAwardTypeTotals));
+                    }
+                }
+            }
+            else {
+                const params = {
+                    filter: {
+                        def_codes: defCodes.map((defc) => defc.code)
                     }
-                });
+                };
+                if (activeTab !== 'all') {
+                    params.filter.award_type_codes = awardTypeGroups[activeTab];
+                } else {
+                    params.filter.award_type = 'assistance';
+                }
+                awardAmountRequest.current = fetchAwardAmounts(params);
+                awardAmountRequest.current.promise
+                    .then((res) => {
+                        setAwardObligations(res.data.obligation);
+                        setAwardOutlays(res.data.outlay);
+                        setNumberOfAwards(res.data.award_count);
+
+                        /* eslint-disable camelcase */
```

Comment:
ditto per above we can use object de-structuring to pass this test and we probably should to make it more readable as to what these properties on the api actually are.